sbteclipse-core/src/main/scala/com/typesafe/sbteclipse/core/EclipsePlugin.scala

```diff
@@ -225,7 +225,7 @@ trait EclipsePlugin {
 
   object EclipseProjectFlavor extends Enumeration {
 
-    val Scala = Value
+    val ScalaIDE = Value
```

Comment:
Yeah, should be 4.0.0 I guess, my bad. I'm way behind on remember what version sbteclipse is.